### ISSUE TITLE
Screen Readers does not read value changed via keyboard when popup is closed

### ISSUE THREAD

**Issue by jivanova** [telerik/kendo-ui-core#3789]
### Bug report

[demo](http://demos.telerik.com/kendo-ui/dropdownlist/index)

### Reproduction of the problem
1. Navigate to the dropdown via TAB key
2. Press down arrow to change the selected value of the dropdownlist

### Current behavior
Value is changed, but the screen reader does not announce the change.The issue stems from the display:none and aria-hidden attributes set to the list when closed. Changing the selected value via keyboard will correctly set the aria-activedescendant attribute to the element, but as its parent has those attributes - the screen reader will not read anything. 

### Expected/desired behavior
Changed value is read by the screen reader

### Environment

* **Kendo UI version:** 2017.3.1026
* **Browser:** all
